docs/content/dagster-plus/managing-deployments/branch-deployments/change-tracking.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "Branch Deployment Change Tracking in Dagster+ | Dagster Docs"
+description: "Change tracking in Branch Deployments makes it eaiser for you and your team to identify how changes in a pull request will impact data assets."
+---
+
+# Branch Deployment Change Tracking in Dagster+
+
+<ApplicableDagsterProduct />
+
+Branch Deployments Change Tracking makes it eaiser for you and your team to identify how changes in a pull request will impact data assets. By the end of this guide, you'll understand how Change Tracking works and what types of asset changes can be detected.
+
+---
+
+## How it works
+
+Branch Deployments compare asset definitions in the branch deployment against the asset definitions in the main deployment. The UI will then mark changed assets accordingly. For example, if the pull request associated with the branch deployment adds a new asset, the UI will display a label indicating the addition.
+
+You can also apply filters to show only new and changed assets in the UI. This makes it easy to understand which assets will be impacted by the changes in the branch deployment.
+
+---
+
+## Supported change types
+
+Change Tracking can detect the following changes to assets:
+
+- [New assets](#new-assets)
+- [Code versions](#code-versions)
+- [Upstream dependencies](#upstream-dependencies)
+- [Partitions definitions](#partitions-definitions)
+
+### New assets
+
+If an asset is new in the branch deployment, the asset will have a **New in branch** label in the asset graph:
+
+<Image
+src="/images/dagster-cloud/managing-deployments/change-tracking-new.png"
+width={764}
+height={308}
+/>
+
+### Code versions
+
+If using the `code_version` argument on the asset decorator, Change Tracking can detect when this value changes.
+
+Some Dagster integrations, like `dagster-dbt`, automatically compute code versions for you. For more information on code versions, including strategies for how to automatically compute code versions, refer to the [Code versioning guide](/guides/dagster/asset-versioning-and-caching).
```

Comment:
realizing that i put the "including strategies for how to automatically compute code versions" line in there optimistically in the hopes that we would write that content. Not sure if that's going to get done by launch, so let's remove it for now

Reply:
Probably not - I'll add a ticket to do a fast follow right after though